include/meta/topics/topic_model.h

```diff
@@ -57,7 +57,8 @@ class topic_model
      * @param k The number of words to return
      * @return the top k most probable words in the topic
      */
-    std::vector<term_prob> top_k(topic_id tid, std::size_t k = 10) const;
+    template <typename T>
+    std::vector<term_prob> top_k(topic_id tid, std::size_t k, T score) const;
```

Comment:
Please take T by universal reference (`T&& score`). We don't want to copy the scorers that have a lot of internal state.